tools/x86_trace_generator/trace_generator.cpp

```diff
@@ -1097,6 +1099,7 @@ void finish(void)
     // thread_id thread_inst_start_count (relative to main thread)
     configFile << thread_info[ii].thread_id << " " << thread_info[ii].inst_count << endl;
   }
+  configFile << t_gen_ver << endl;
```

Comment:
can you move the t_getn_ver number before the for-loop? we want to check the version number as early line as possible.